vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -122,19 +123,13 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
     }
 
     private openHandler(xhr: XMLHttpRequestInstrumented, method, url, async) {
-        let id: string;
-        if (this._context && this._context.telemetryTrace && this._context.telemetryTrace.traceID) {
+        var traceID = (this._context && this._context.telemetryTrace && this._context.telemetryTrace.traceID) ? this._context.telemetryTrace.traceID : Util.generateW3CId();
```

Comment:
`(this._context && this._context.telemetryTrace && this._context.telemetryTrace.traceID) || Util.generateW3CId();`